examples/compute_instance/simple/variables.tf

```diff
@@ -35,6 +35,16 @@ variable "num_instances" {
 }
 
 variable "service_account" {
-  type        = "map"
-  description = "Service account email address and scopes"
+  default = null
+  type        = list(object({
+        email = string,
+       scopes = set(string)
```

Comment:
Are you sure you don't need to use list(string)?

Reply:
I'm sure because in the source code of google provider this variable is set of strings